homeassistant/components/http.py

```diff
@@ -221,28 +220,6 @@ def send(self, request, filepath):
 _GZIP_FILE_SENDER = GzipFileSender()
 
 
-class HAStaticRoute(StaticRoute):
-    """StaticRoute with support for fingerprinting."""
-
-    def __init__(self, prefix, path):
-        """Initialize a static route with gzip and cache busting support."""
-        super().__init__(None, prefix, path)
-        self._file_sender = _GZIP_FILE_SENDER
-
-    def match(self, path):
-        """Match path to filename."""
-        if not path.startswith(self._prefix):
-            return None
-
-        # Extra sauce to remove fingerprinted resource names
```

Comment:
@balloob you're right, it does, which is a misnomer in my explanation.

I believe that the HA start event is being fired _before_ the `frontend` setup is called (because of the GIL). Which means that `http` will start the server before frontend is ready.